controllers/users.js

```diff
@@ -65,7 +67,7 @@ const login = (req, res, next) => {
 
   return User.findUserByCredentials(email, password)
     .then(user => {
-      const token = jwt.sign({ _id: user._id }, NODE_ENV === 'production' ? JWT_SECRET : 'dev-secret');
+      const token = jwt.sign({ _id: user._id }, NODE_ENV === 'production' ? JWT_SECRET : SECRET);
       res.cookie('jwt', token, { maxAge: 3600000 * 24 * 7, httpOnly: true }).end();
     })
     .catch(() => next(new UnauthorizedError('Неправильные почта или пароль')));
```

Comment:
Можно лучше: избавиться от фиксирования возвращаемой ошибки при обработке ошибок.
Если метод `findUserByCredentials()` отклонит обещание по причине неправильной пары логин/пароль, то вернётся 401:
https://github.com/romcath/praktikum_backend-mesto/blob/15990bb6c12ac89f0efba7e3319600f2546f3658/models/user.js#L45-L48
Тоже самое произойдёт, если пользователь не был найден в базе данных (конечно если в `orFail()` всё-таки будет добавлена отправка `UnauthorizedError`).
Ну а ошибки типа `SyntaxError`, `TypeError` обрабатываются централизованным обработчиком ошибок. Поэтому можно:
```suggestion
    .catch(next);
```